waspc/src/Wasp/Generator/WebAppGenerator/AuthG.hs

```diff
@@ -52,7 +52,7 @@ genLogout = return $ C.mkTmplFd (C.asTmplFile [relfile|src/auth/logout.js|])
 genCreateAuthRequiredPage :: AS.Auth.Auth -> Generator FileDraft
 genCreateAuthRequiredPage auth =
   compileTmplToSamePath
-    [relfile|auth/pages/createAuthRequiredPage.js|]
+    [relfile|auth/pages/createAuthRequiredPage.jsx|]
```

Comment:
Vite requires that all of the files that use `JSX` have the `.jsx` extension.

Reply:
I don't think this should be a big problem, but we will have to label it a breaking change and users would need to change file extensions during the upgrade (but the work will be worth it! :D).